src/components/SplitView/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { ToggleButton, TOGGLE_BUTTON_WIDTH } from './ToggleButton';
+
+const FULL_HEIGHT = { height: '100%' };
+const DETAIL_VIEW_STYLE = { marginLeft: TOGGLE_BUTTON_WIDTH };
+
+export class SplitView extends Component {
+  static propTypes = {
+    assistiveTextOpen: PropTypes.string,
+    assistiveTextClose: PropTypes.string,
+    className: PropTypes.string,
+    detail: PropTypes.node.isRequired,
+    id: PropTypes.string.isRequired,
+    initialIsOpen: PropTypes.bool,
+    master: PropTypes.node.isRequired,
+    masterMaxWidth: PropTypes.string,
+    masterMinWidth: PropTypes.string,
+    onOpen: PropTypes.func,
+    onClose: PropTypes.func,
+  };
+  static defaultProps = {
+    assistiveTextOpen: '',
+    assistiveTextClose: '',
+    className: null,
+    initialIsOpen: false,
```

Comment:
Default should be `true` imho

Reply:
It is strong antipattern to have 'true' default values for boolean props

Reply:
it breaks semantics of using them like normal html tags:
```
<SplitView initialIsOpen />
```

Reply:
I agree about it being an antipattern, `initialIsIsOpen` still seems like the default case though. Would it make sense to use `initialIsClosed` prop here instead then?